_infra/helm/frontstage/values.yaml

```diff
@@ -24,6 +24,7 @@ ingress:
   surveysHost: surveys.example.com
   certNameSurveys: surveys-cert
   timeoutSec: 30
+  sslPolicy: frontstage-ingress-ssl-policy
```

Comment:
Do we want to set this in the config? Or was this enough as they're all going to have the same name?

Reply:
This policy will have the same name in all environments (including prod and preprod). Dev doesn't have an SSL policy (since there's no ingress set up), and even if it did, the SSL policy wouldn't be namespace specific since there'd be only one policy set up for the ras-rm-dev environment as a whole.